Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.13
+RUN apk add cmatrix
+CMD [ "cmatrix" ]
+
+#### Litle doc ####
+# Now with command `docker build -t <NameOfImage:Tag> .` An image will create that the main work is to run Cmatrix 
+# After creation with command `docker run -it <NameOfImage:Tag>` Cmatrix will run :)
+
+#### For @abishekvashok ####
+# at first wanted to install from source code but can't autoconf in alpine and this way is neater 
```

Comment:
Hey, I feel the source code installation makes more sense. I don't know much about docker, but I can see an autoconf package for alpine available here: https://pkgs.alpinelinux.org/package/v3.3/main/x86/autoconf

Reply:
Your Welcome , 
Yes absolutely , installing from source code is best and essential but we add DockerFile for people who want to see a preview of the code and don't want to install in Host OS computer so they run docker and see the code in a container .

I use linux alpine because (because it is a 1.82MB linux :D) and it fits for everyone that has an unstable network who can't download Ubuntu image (about 30 - 50 MB) And also I see the Alpine link but the best way for docker to use an image is to get it from [hub.docker](https://hub.docker.com/).

OK, of course I add doc in commit body :)

Reply:
50 mb seems reasonable if we can move the step of compilation there. We can use a release which has a configure so we won't need autotools.